app/src/main/res/values-land/dimens.xml

```diff
@@ -307,4 +307,10 @@
   <dimen name="promoted_story_card_layout_margin_start">8dp</dimen>
   <dimen name="promoted_story_card_text_size">14sp</dimen>
   <dimen name="promoted_story_card_margin_top">8dp</dimen>
+
+  <!-- Topic Revision Fragment -->
+  <dimen name="topic_revision_fragment_paddingStart">56dp</dimen>
```

Comment:
Rename this and all other dimens name to `topic_revision_fragment_paddingStart `.
Basically all names should contain only small-characters separated by `_`.